modules/surface_matching/src/ppf_match_3d.cpp

```diff
@@ -583,9 +583,7 @@ void PPF3DDetector::match(const Mat& pc, std::vector<Pose3DPtr>& results, const
       poseList.push_back(pose);
     }
 
-#if defined (_OPENMP)
     free(accumulator);
```

Comment:
> uint* accumulator = (uint*)calloc(numAngles*n, sizeof(uint));

BTW, it is better to avoid manual memory management and use `cv::AutoBuffer` or `std::vector`.

Reply:
Totally agree.

Unfortunately I found this bug while helping a friend, not using this opencv module myself I do not feel confident enough to refactor the code to a proper "C++ style" memory management.